### ISSUE TITLE
Possible to use with one modality? What are the data size limits?

### ISSUE THREAD

**Issue by vitkl** [Center-for-Health-Data-Science/multiDGD#4]
Hi @viktoriaschuster 

Congratulations on getting this to a paper stage! Excited to try on my data. A few questions:

1. Is the package possible to use with one modality?
2. What are the data size limits? 
A) The number of cells. Can I use 100k or 600k cells?
B) The number of DNA regions in ATAC modality - this value is commonly very large eg 300-600k regions or even 1mln regions.
3. Have you tried applying to data with 100-500 distinct cell populations? How should the hyperparameters of the decoder and latent space modified to accommodate?

**Comment by viktoriaschuster** [comment]
Hi @vitkl,

Thanks a lot! To answer your questions:

1. It is designed to be flexible in terms of number of modalities, but I haven't tested it yet. The best setup is to either use a mudata object or an anndata object with a var column for 'modality'. There is also my older package scDGD (https://github.com/Center-for-Health-Data-Science/scDGD) for single cell RNA counts. All of this is still in development, so I am still working on harmonizing these two.
2. So far, we have worked with up to 100k cells and ~300k features. Based on this and previous work, I am expecting multiDGD to scale better with number of features than MultiVI, but MultiVI performing better when reaching 1M cells with <100k features.
3. The data sets I have applied so far only went up to ~50 cell types. In such a case as you are describing, it really depends on what level of clustering you would like to get. If you want these many cell populations to be as stringently separated as possible, go with the default setting of number of Gaussian Mixture components (taken from the 'observable' in the data set) and maybe set the scale of the mean prior to something larger ('softball_scale' > 2).

I see that it is currently hard to use the model with any customization. I will shortly add a preliminary documentation :)
